manifests/server.pp

```diff
@@ -50,6 +53,13 @@
       require     => Package['rsync'],
     }
   } else {
+    if defined('$rsyncdpackage'){
```

Comment:
Rather than relying on "defined" can we implement a specific logic here ? Usage of the separate package is now specific to CentOS/RHEL so a single block logic is simpler. We can revisit this when any other distro follows that direction.